dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/application/usecase/DispatcherUserCase.java

```diff
@@ -75,6 +65,43 @@ public void triggerDispatching() {
         log.info("Finished dispatching");
     }
 
+    /**
+     * Process all files inside a folder recursively.
+     * See {@link DispatcherUserCase#processFile(UseCase, File)}.
+     *
+     * @param useCase The bucket of the folder.
+     * @param folder The full path of the folder.
+     * @return Error which occurred during processing (Key: file path, value: error).
+     */
+    private @NotNull
+    Map<String, Throwable> processDirectory(final UseCase useCase, final String folder, final boolean recursive) {
+        final List<File> readyFiles = fileSystemOutPort.getMatchingFiles(
+                useCase.getBucket(),
+                folder,
+                recursive,
```

Comment:
Werden bei `recursive=true` nicht Dateien in Unterverzeichnissen mehrmals verarbeitet?

Also bei
dir_a
____dir_b
________file_a

wird in dem Beispiel nicht file_a einmal bei der Verarbeitung von dir_a (rekursiv/transitiv) und einmal bei der Verarbeitung von dir_b (direkt) verarbeitet?

Reply:
`dir_a` is called with recursive false and only the subdirs are searched recursively.